boards/pyboard/Makefile.features

```diff
@@ -0,0 +1,13 @@
+# Put defined MCU peripherals here (in alphabetical order)
+#FEATURES_PROVIDED += periph_dma
```

Comment:
There is no configuration provided for DMA, so I think this should be removed. Same for PWM.

Reply:
done